Content.Shared/Humanoid/NamingSystem.cs

```diff
@@ -35,6 +35,9 @@ public string GetName(string species, Gender? gender = null)
                 case SpeciesNaming.FirstDashFirst:
                     return Loc.GetString("namepreset-firstdashfirst",
                         ("first1", GetFirstName(speciesProto, gender)), ("first2", GetFirstName(speciesProto, gender)));
+                case SpeciesNaming.LastFirst:
+                    return Loc.GetString("namepreset-lastfirst",
+                        ("last", GetLastName(speciesProto)), ("first", GetFirstName(speciesProto, gender)));
```

Comment:
This should have a comment on it since it's an upstream file, so it won't be overwritten by an update

Something like "this is for [species]"